website/api/zkvm/composition.md

```diff
@@ -0,0 +1,43 @@
+# Proof Composition
+
+RISC Zero's zkVM is the only zkVM that offers a convenient API for efficiently verifying proofs inside the guest program.
```

Comment:
This is somewhat of a tenuous statement, as SP1 provide a reasonable API and I'm sure others will in the future. Maybe we just talk about how great composition is without comparing it to others.

https://github.com/succinctlabs/sp1/blob/main/examples/aggregation/script/src/main.rs